swap/swap_test.go

```diff
@@ -136,6 +136,124 @@ func testBalances(t *testing.T, swap *Swap, expectedBalances map[enode.ID]int64)
 	}
 }
 
+type storeKeysTestCases struct {
+	nodeID                    enode.ID
+	expectedBalanceKey        string
+	expectedSentChequeKey     string
+	expectedReceivedChequeKey string
+}
+
+func TestStoreKeys(t *testing.T) {
```

Comment:
A test should have a description, at least a comment what it is supposed to do.
A PR without commented exported function names will be rejected

Reply:
Fixed, thanks.